test/v2_integration.py

```diff
@@ -1005,7 +1005,7 @@ def test_http_multiva_threshold_pass():
 
     # Configure a guestlist that will pass the multiVA threshold test by
     # allowing the primary VA at some, but not all, remotes.
-    guestlist = {"boulder": 1, "boulder-remoteva-a": 1, "boulder-remoteva-b": 1, "remoteva-a": 1}
+    guestlist = {"boulder": 1, "boulder-remoteva-a": 1, "boulder-remoteva-b": 1, "remoteva-c": 1, "remoteva-a": 1}
```

Comment:
Looks like this is maybe a spot where I missed removing the "boulder-remoteva-x" entries in https://github.com/letsencrypt/boulder/pull/7787. I think the second and third entries here should probably be removed, and then the "remoteva-x" entries should be sorted alphabetically.